packages/flutter/lib/src/gestures/binding.dart

```diff
@@ -421,6 +434,11 @@ mixin GestureBinding on BindingBase implements HitTestable, HitTestDispatcher, H
   /// Non-negative [samplingOffset] is allowed but will effectively
   /// disable resampling.
   Duration samplingOffset = _defaultSamplingOffset;
+
+  /// Offset relative to current sample time that should be used to
+  /// estimate next sample time. The [samplingInterval] is expected to
+  /// be non-negative.
+  Duration samplingInterval = _defaultSamplingInterval;
```

Comment:
I'm a little concerned of adding this to the public API since it will be hard to remove in the future if we decided to hide it and let the engine or some internal system to decide its value. Can we keep it private for now as we're just using 16ms in current use cases?

Reply:
Yep, done.